### ISSUE TITLE
Excluded and Test files readability issue

### ISSUE THREAD

**Issue by kawazoe** [subtheme-dev/monokai-pro#45]
**Describe the bug**
Files marked as *excluded* and to a certain point *test* in the project view are completely very difficult to read, especially when they are also ignored in the repository.

This is specially bad when using the Octagon filter which dims the text when compared to the default theme.

I would suggest to take inspiration from the New UI's Dark theme and One Dark and pick more saturated and darker colors for the red (excluded) and green (test) highlights to match the dimmer text of Monokai.

**Screenshots**
![image](https://user-images.githubusercontent.com/406575/202816392-2253c0d0-38ff-4b45-957a-503d0a0ff503.png)
![image](https://user-images.githubusercontent.com/406575/202816667-e826ac48-eb2f-4ac2-a856-4e18b9944ea8.png)

**Comment by MikeDarksky** [comment]
Yes! I was going to raise the same issue. Looking for something in node_modules using the Spectrum variant is also pretty bad. Update would definitely be appreciated.

**Comment by ForsakenHarmony** [comment]
This is not by design, I think this is a bug

WebStorm
![](https://i.hrmny.sh/OdHVPhNADG.png)

CLion
![](https://i.hrmny.sh/ggNApCq7zA.png)

**Comment by coderlzw-cn** [comment]
It's 2024, isn't this bug ready to be fixed?
